### ISSUE TITLE
🌟 [EPIC] Accessibility

### ISSUE THREAD

**Issue by sushidave** [Human-Connection/Human-Connection#3633]
## :rocket: Feature

Make the network accessible according to the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/).

### User Problem

Handicapped people e.g. with physical or cognitive disabilities cannot use the network to its full extent. This issue solves the problem.

### Implementation

**Help wanted:**

- People who add the guidelines to the documentation (for improvement and future development).
- People who know about the WCAG guidelines, assess the existing network and identify rooms for improvement.
- People who propose solutions.
- People who code it.
- People who test it.

**Steps:**

1. Documentation:
- [ ] Include accessibility in documentation

2. Requirements:
- [ ] Identify what needs to be improved
- [ ] Propose solutions

3. Code implememtation:
- [ ] Registration, login, logout
- [ ] Profile page
- [ ] Account settings
- [ ] Newsfeed
- [ ] Search
- [ ] Filters
- [ ] Single post
- [ ] Post editor
- [ ] Comments
- [ ] Notifications
- [ ] Moderation
- [ ] Administration
- [ ] any others?

### Design & Layout

To be assessed, see point 2 above.

### Validation

Adhere to the WCAG standards. 

Benefits due to accessibility and inclusion: 
- More users able to use the network. 
- Less support issues. 
- Higher user satisfaction. 
- Promote accessibility as a USP feature of this network.

### Additional context

As discussed with @Raimundo-dev yesterday.
